chemclipse/plugins/org.eclipse.chemclipse.xxd.model/src/org/eclipse/chemclipse/xxd/model/support/ProcessPeaksByAreaFilterLocalSettings.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Lablicate GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Alexander Stark - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chemclipse.xxd.model.support;
+
+import org.eclipse.chemclipse.xxd.model.filter.peaks.ProcessPeaksByIntegratedAreaFilterSettings;
+import org.eclipse.chemclipse.xxd.model.filter.peaks.ProcessPeaksByPercentageAreaFilterSettings;
+
+public class ProcessPeaksByAreaFilterLocalSettings {
+
+	private double[] areaLimits = new double[] {0.0, 0.0};
```

Comment:
Why is this an arrary? Please add javadoc to variables, where it is not perfectly clear what they mean.

Reply:
In the meantime I re-factored this to
`private double lowerLimit = 0.0d;
private double upperLimit = 0.0d;`
 -> please refer to newer commits. I will be happy to add a Javadoc.